packages/react-hooks/use-scroll/useScroll.ts

```diff
@@ -0,0 +1,42 @@
+import { useCallback } from "react";
+
+export type UseScrollParams = {
+  elementRef: React.RefObject<HTMLElement>;
+};
+
+export enum ScrollDirections {
```

Comment:
could you change this type to use sum types instead of enums? Enums are less recommended in TS

Reply:
Sounds good!

Reply:
New changes were added. Please review :raised_hands: